apps/etterlatte-hendelser-pdl/src/main/kotlin/hendelserpdl/leesah/PersonHendelseFordeler.kt

```diff
@@ -10,20 +10,30 @@ import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import java.time.format.DateTimeFormatter
 
+val sikkerLogg: Logger = LoggerFactory.getLogger("sikkerLogg")
+
 class PersonHendelseFordeler(
     private val postHendelser: ILivsHendelserRapid,
     private val pdlService: Pdl
 ) {
-    private val log: Logger = LoggerFactory.getLogger(PersonHendelseFordeler::class.java)
+    private val logger: Logger = LoggerFactory.getLogger(PersonHendelseFordeler::class.java)
 
     suspend fun haandterHendelse(personhendelse: Personhendelse) {
-        when (personhendelse.opplysningstype) {
-            LeesahOpplysningstyper.DOEDSFALL_V1.toString() -> haandterDoedsendelse(personhendelse)
-            LeesahOpplysningstyper.UTFLYTTING_FRA_NORGE.toString() -> haandterUtflyttingFraNorge(personhendelse)
-            LeesahOpplysningstyper.FORELDERBARNRELASJON_V1.toString() -> haandterForelderBarnRelasjon(personhendelse)
-            LeesahOpplysningstyper.ADRESSEBESKYTTELSE_V1.toString() -> haandterAdressebeskyttelse(personhendelse)
-            LeesahOpplysningstyper.VERGEMAAL_ELLER_FREMTIDSFULLMAKT_V1.toString() -> haandterVergemaal(personhendelse)
-            else -> log.info("Så en hendelse av type ${personhendelse.opplysningstype} som vi ikke håndterer")
+        if (personhendelse.personidenter.first() == null) {
```

Comment:
```suggestion
        if (personhendelse.personidenter.isEmptyOrNull()) {
```
med mindre de har lagt til null i lista si da 🤠

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1446/commits/b9a741fa250675ab49249ff5ff0808be69964142